lib/core.js

```diff
@@ -23,6 +23,16 @@ const loadOptionsInConfigFileIfExists = async (path, options = {}, componentName
 	return options
 }
 
+const loadReacliConfigurationFileIfNotIgnored = async (ignoreConfigFile, path, options, componentName = "") => {
```

Comment:
I think the function's name gives too much details about what it does. In my opinion, `loadReacliConfiguration` is enough and ignoreConfigFile could be added as an optional parameter at the end as such:

`loadReacliConfiguration = async (path, options, componentName = "", ignoreConfigFile)`

Reply:
https://github.com/reacli/cli/pull/48/commits/b221ae0995396f63c2aca2af55e80e10ee2083e5